src/viewer.js

```diff
@@ -1608,6 +1611,20 @@ $.extend( $.Viewer.prototype, $.EventSource.prototype, $.ControlDock.prototype,
         return this.world.removeItem(drawer);
     },
 
+    /**
+     * Update pixel density ration, clears all tiles and triggers updates for
```

Comment:
This comment should mention the fact that it only clears the tiles if the ratio has changed. Also, you've got a typo: `ration`.

Reply:
Thanks. I have fixed it.